README.md

```diff
@@ -398,18 +398,53 @@ end
 ```
 
 ### sensu_entity
-An entity is a representation of anything that needs to be monitored. It can be either an `agent` or a `proxy`.
+An entity is a representation of anything that needs to be monitored.
 
 #### Properties
-* `subscriptions` An array of subscriptions. If no subscriptions are provided,
-it defaults to an entity-specific subscription list: `[entity:{ID}]`.
-* `entity_class` **required** the entity type, must be either `agent` or `proxy`.
+* `entity_class` **required** the entity type, should be either `agent` or `proxy`. Entities should only be created as `agent` in this manner if unable to deploy them through the `sensu_agent` resource on the machine.
```

Comment:
I think this is a copy, paste, typo?
```suggestion
* `entity_class` **required** the entity type, should be either `agent` or `proxy`. Entities should only be created as `agent` in this manner if unable to deploy them through the `sensu_check` resource on the machine.
```

Reply:
No, this is correct. The `sensu_agent` resource is intended to deploy the sensu-agent and that automatically creates entities. The only reason why we'd want to create an entity as an agent through sensu_entity would be if, for some very obscure reason, the sensu agent was already installed, not running, and not able to reach out to the backend. I was honestly tempted to just say that this should just be `proxy` only, since I can't think of a single good reason to create agent entities this way.

If an entity is created through `sensu_check`, it's going to be created as a proxy entity, not an agent.

Reply:
@majormoses Is the explanation unclear, or anything else I need to change?